vscode-extension/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "azps-tools",
 	"displayName": "Azure PowerShell Tools",
 	"description": "(preview) Automate the migration of Azure PowerShell scripts and modules from AzureRM to the Az PowerShell module.",
-	"version": "0.1.0",
+	"version": "0.1.1",
```

Comment:
@chenlei-yang , please check if extension version needs to be updated?

Reply:
@dingmeng-xue Yes, we need to change the target version in the language server, and I already committed it.